sciLifeLab_utils/run_delivery.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import glob
+import re
+import subprocess
+
+def main(arg):
+    project = os.path.split(os.path.realpath(arg.source))[1]
+    move_from_path = "{}/*/results/".format(arg.source)
+    pathway = glob.glob(move_from_path)
+    pattern = re.compile(r"/[^/]+/[^/]+/[^/]+/[^/]+/[^/]+/([^/]+)/") #Matches the group betwen the 6th / /
```

Comment:
>> Also, if these are just paths and you just want to get the last folder in the path, could you use os.path.basename ?
>> 

I want the name of the sample to use later in the script. 
For example: /proj/b2013064/nobackup/remi/A.Test_16_01/P0123_101/results/
the re should return ‘P0123_101’so the second last folder. Is there an easy way to do that with os.path?

> On 26 Jan 2016, at 14:10, Denis Moreno <notifications@github.com> wrote:
> 
> In sciLifeLab_utils/run_delivery.py <https://github.com/SciLifeLab/NouGAT/pull/35#discussion_r50832145>:
> 
> > @@ -0,0 +1,45 @@
> > +import argparse
> > +import os
> > +import glob
> > +import re
> > +import subprocess
> > +
> > +def main(arg):
> > +    project = os.path.split(os.path.realpath(arg.source))[1]
> > +    move_from_path = "{}/*/results/".format(arg.source)
> > +    pathway = glob.glob(move_from_path)
> > +    pattern = re.compile(r"/[^/]+/[^/]+/[^/]+/[^/]+/[^/]+/([^/]+)/") #Matches the group betwen the 6th / /
> Also, if these are just paths and you just want to get the last folder in the path, could you use os.path.basename ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SciLifeLab/NouGAT/pull/35/files#r50832145>.
>